src/deviceApiCalls/transports/extension.transport.js

```diff
@@ -69,6 +69,8 @@ async function extensionSpecificRuntimeConfiguration (deviceApi) {
             contentScope: contentScope,
             // @ts-ignore
             userPreferences: {
+                // Copy to user preferences to match schema from CSS
+                locale: contentScope.locale,
```

Comment:
Interestingly, the extension returns all of the defined `userPreferences` in the `contentScope` variable above. This change would technically lead to a duplicate `locale: "en"` property in the extension's response to `getRuntimeConfiguration`, but the cost of that is quite low.

Reply:
agreed - don't worry about a  bit of duplication, since this is an area that will improve later.